README.md

```diff
@@ -21,7 +21,7 @@ Consider this file name:
 What is the purpose of this file, and what is the purpose of the numbers at the beginning of its name?
 
 ```text
-Your answer...
+This is a migration file and the numbers at the beginning are the timestamp of when that migration was ran. The timestamps are there to keep a record of changes you make to your tables.
```

Comment:
The timestamp is for when the migration was created.  We do not know if it has been run yet.  The timestamps are used to make sure migrations run in the appropriate order